src/js/themes/hpe.js

```diff
@@ -1974,7 +1999,16 @@ const buildTheme = (tokens, flags) => {
         ],
         columns: [['small', 'flex'], 'auto'],
         rows: ['auto', 'auto', 'auto'],
-        gap: { row: 'xsmall', column: 'medium' },
+        gap: { row: '3xsmall', column: 'medium' },
+      },
+      small: {
+        gap: { row: '3xsmall', column: 'large' },
+      },
+      medium: {
+        gap: { row: '3xsmall', column: 'medium' },
+      },
+      large: {
+        gap: { row: '3xsmall', column: 'large' },
```

Comment:
Looks like these have a pad defined in base.js in addition to a gap

Reply:
Handled adding the `pad` above ^^ was in a different theme obj than here but I was missing it.